consensus/src/epoch_manager.rs

```diff
@@ -263,9 +251,48 @@ impl EpochManager {
                     vec![1; proposers.len()]
                 };
 
+                // First two epochs are immediatelly skipped after a few transactions, so we don't want to look at their history.
+                let epoch_to_proposers = if use_history_from_previous_epoch && epoch_state.epoch > 2
+                {
+                    self.storage
+                        .aptos_db()
+                        .get_epoch_ending_ledger_infos(epoch_state.epoch - 2, epoch_state.epoch)
```

Comment:
Any reason we are getting history only for last 2 epoch? I assume if every epoch is 1 hour - considering history of multiple epochs will be insightful.  How about we make this configurable instead?